### PR TITLE
Create release notes for v1.0.0-preview.7

### LINKED ISSUE #190: 🚧Create release notes for v1.0.0-preview.7
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_<title_**>' section.

### Description

Create release notes for preview release _**v1.0.0-preview.7**_.

### Acceptance Criteria

**This issue is finished when:**
- [ ] Release notes created

### ToDo Items

- [ ] Draft pull request created and linked to this issue
- [X] Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
- [X] Issue linked to the proper project
- [X] Issue linked to proper milestone

### Issue Dependencies

_No response_

### Related Work

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>

<h2 style="font-weight:bold" align="center">✅Development Checklist✅</h2>

<details open><summary>🌳Branching🌳</summary>

Does the name of the head(source) branch for this pull request have the correct **_preview feature branch_** syntax?

Syntax: _preview/feature/&lt;issue-id&gt;-&lt;description&gt;_  
Example: _preview/feature/123-my-preview-change_
- [x] Yes
- [ ] No

Does the name of the base(target) branch for this pull request have the correct **_preview branch_** syntax?

Syntax: _preview/v&lt;major-number&gt;.&lt;minor-number&gt;.&lt;patch-number&gt;-preview.&lt;prev-number&gt;_  
Example: _preview/v1.2.3-preview.4_
- [x] Yes
- [ ] No

Is the head(source) branch for this pull request created from a branch with the correct **_preview branch_** syntax?

Syntax: _preview/v&lt;major-number&gt;.&lt;minor-number&gt;.&lt;patch-number&gt;-preview.&lt;prev-number&gt;_  
Example: _preview/v1.2.3-preview.4_
- [x] Yes
- [ ] No
</details>

<details open><summary>🐛Bugs🐛</summary>

Contains Bug Fix(es)
- [ ] Yes
  - [ ] A bug-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>💣Breaking Change(s)💣</summary>

Contains [breaking change(s)](https://docs.microsoft.com/en-us/dotnet/core/compatibility/#modifications-to-the-public-contract)
- [ ] Yes
  - [ ] A breaking-change-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>✨Enhancements✨</summary>

Contains enhancement(s) that add a feature or behavior.
- [ ] Yes
  - [ ] An enhancement-label label has been added to the pull request.
- [x]  No
</details>

<details open><summary>⚙️Workflow (CI/CD) Changes⚙️</summary>

Contains changes to workflow files. These changes can only be done by the project maintainers.
- [ ] Yes
  - [ ] A workflow-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>📃Documentation Updates📃</summary>

Contains changes that require updates to existing **_code_** and/or **_Velaptor product_** documentation.
- [x] Yes
  - [x] I have updated the documentation accordingly.
    - Choose the type of documentation.
    - [ ] Code Documentation
      - [ ] A documentation-code-label label has been added to the pull request.
    - [x] Product Documentation
      - [x] A documentation-product-label label has been added to the pull request.
- [ ] No
</details>

<details open><summary>🧪Unit Testing🧪</summary>

My change requires unit tests to be written.
- [ ] Yes
- [x] No
</details>

<details open><summary>🧪Manual Testing🧪</summary>

I have manually tested my changes using the project named **_VelaptorTesting_**.
- [ ] Yes
- [x] No
</details>

<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] Pull request title matches the title of the linked issue.
- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.
- [x] My code follows the coding style of this project.
  - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
- [x] A preview-label label has been added to the pull request.
- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+<h2 align="center" style='font-weight:bold'>Quick Reminder</h2>
+
+As with all software, including releases, there is always a chance for issues and bugs.  It is also common to miss changes in the release notes when there are many.  This is even more common in preview releases.
+
+---
+
+<h2 style="font-weight:bold" align="center">New 🎉</h2>
+
+1. Add ability to render rectangles to the scene with the various attributes below
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:11]: Grammar
```suggestion
1. Add ability to render rectangles to the scene with the various attributes below:  
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   - Ability to change the fill color of a filled rectangle
+   - Ability to change the border color of an empty rectangle
+   - Ability to have a horizontal color gradient with a start and end color
+     - This can be applied to filled and empty rectangles
+   - Ability to have a vertical color gradient with a start and end color
+     - This can be applied to filled and empty rectangles
+   - Ability to change the border thickness of an empty rectangle
+   - Ability to set radius of each corner independently
+     - This can be applied to filled and empty rectangles
+2. Loading sounds can not be cached to greatly improve sound loading performance.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:24]: Spelling
```suggestion
2. Loading sounds cannot be cached to greatly improve sound loading performance.  
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+2. Loading sounds can not be cached to greatly improve sound loading performance.  
+   GitHub Issues: [#131](https://github.com/KinsonDigital/Velaptor/issues/131)
+3. Loading sounds can now be loaded via a fully qualified file path.  This means you do not have to only load sounds from the content system!!  
+   GitHub Issues: [#128](https://github.com/KinsonDigital/Velaptor/issues/128)
+
+---
+
+<h2 style="font-weight:bold" align="center">Breaking Changes 💣</h2>
+
+1. The class `SystemMonitor` constructor used to require an `IPlatform` type argument but not does not and has no arguments.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:33]: Grammar
```suggestion
1. The class `SystemMonitor` constructor no longer requires an `IPlatform` type argument and has no arguments.  
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+<h2 style="font-weight:bold" align="center">Breaking Changes 💣</h2>
+
+1. The class `SystemMonitor` constructor used to require an `IPlatform` type argument but not does not and has no arguments.  
+   GitHub Issues: [#153](https://github.com/KinsonDigital/Velaptor/issues/153)
+
+---
+
+<h2 style="font-weight:bold" align="center">Improvements 🌟</h2>
+
+1. Upgraded all projects in solution to use C# language version **_v10_**
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:40]: Do you need to name the solution? 
Do you need to write out "version?" you already have v10. Seems redundant.

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+1. The class `SystemMonitor` constructor used to require an `IPlatform` type argument but not does not and has no arguments.  
+   GitHub Issues: [#153](https://github.com/KinsonDigital/Velaptor/issues/153)
+
+---
+
+<h2 style="font-weight:bold" align="center">Improvements 🌟</h2>
+
+1. Upgraded all projects in solution to use C# language version **_v10_**  
+   GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
+2. Upgraded all projects in solution to use **_NET 6.0_**
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:42]: Does the solution need to be named?
Should this be .Net 6.0?

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+<h2 style="font-weight:bold" align="center">Improvements 🌟</h2>
+
+1. Upgraded all projects in solution to use C# language version **_v10_**  
+   GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
+2. Upgraded all projects in solution to use **_NET 6.0_**  
+   GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
+1. Improved constructor argument null checks by implementing a guarding patterns
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:44]: Grammar
```suggestion
1. Improved constructor argument null checks by implementing a guarding pattern  
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+<h2 style="font-weight:bold" align="center">Improvements 🌟</h2>
+
+1. Upgraded all projects in solution to use C# language version **_v10_**  
+   GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
+2. Upgraded all projects in solution to use **_NET 6.0_**  
+   GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
+1. Improved constructor argument null checks by implementing a guarding patterns  
+   GitHub Issues: [#153](https://github.com/KinsonDigital/Velaptor/issues/153)
+   - This also resulted in null checks that were not being performed previously
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:46]: SS
```suggestion
   - This resulted in null checks that were not being performed previously
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+<h2 style="font-weight:bold" align="center">Other 👏</h2>
+
+1. Improvements to PR templates such as the code review checklist, grammar, and other misc. items.  This was done on all pull request templates.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:52]: Grammar
```suggestion
1. Improvements to PR templates such as the code review checklist, grammar, and other miscellaneous items.  This was done on all pull request templates.  
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+<h2 style="font-weight:bold" align="center">Other 👏</h2>
+
+1. Improvements to PR templates such as the code review checklist, grammar, and other misc. items.  This was done on all pull request templates.  
+   GitHub Issues: [#150](https://github.com/KinsonDigital/Velaptor/issues/150), [#158](https://github.com/KinsonDigital/Velaptor/issues/158)
+2. Simple graphical fix with light and dark mode versions of the branching diagram.  
+   GitHub Issues: [#159](https://github.com/KinsonDigital/Velaptor/issues/159)
+3. Add a **_BACKERS.md_** file to the root of the solution to hold a list of any backers that
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:56]: Grammar
```suggestion
3. Added a **_BACKERS.md_** file to the root of the solution to hold a list of any backers that  
```
Incomplete sentence.  "Any backers that...."

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
+<h1 align="center" style='color:mediumseagreen;font-weight:bold'>Velaptor Preview Release Notes - v1.0.0-preview.7</h1>
+
+<h2 align="center" style='font-weight:bold'>Quick Reminder</h2>
+
+As with all software, including releases, there is always a chance for issues and bugs.  It is also common to miss changes in the release notes when there are many.  This is even more common in preview releases.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:5]: Put periods on the end of all sentences. 😘

**Review by kselena — CHANGES_REQUESTED** [review]

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md (hunk 3 above)
Comment (CalvinWilkinson) [reply Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:33]: Upon thinking about this one, i think that it is better like this.

Thoughts?

```suggestion
1. The class `SystemMonitor` constructor no longer requires an `IPlatform` type argument.
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md (hunk 4 above)
Comment (CalvinWilkinson) [reply Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:40]: It does not matter to me.  I see what your saying here.
It is common to prefix a version with the letter `v` and can be interpreted by people as `version`.  I can remove it.  😀

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md (hunk 5 above)
Comment (CalvinWilkinson) [reply Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:42]: It is common to use .NET with all caps.
The solution does not need to be named.  There is only one solution and 90% of the time, projects only have one solution even though you can have more.

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md (hunk 10 above)
Comment (CalvinWilkinson) [reply Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:5]: 😚

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 11]
```diff
@@ -21,39 +21,39 @@ As with all software, including releases, there is always a chance for issues an
    - Ability to change the border thickness of an empty rectangle
    - Ability to set radius of each corner independently
      - This can be applied to filled and empty rectangles
-2. Loading sounds can not be cached to greatly improve sound loading performance.  
+2. Loading sounds cannot be cached to greatly improve sound loading performance.  
    GitHub Issues: [#131](https://github.com/KinsonDigital/Velaptor/issues/131)
-3. Loading sounds can now be loaded via a fully qualified file path.  This means you do not have to only load sounds from the content system!!  
+3. Loading sounds can now be loaded via a fully qualified file path.  This means you do not have to only load sounds from the content system!!.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:26]: Remove period

```suggestion
3. Loading sounds can now be loaded via a fully qualified file path.  This means you do not have to only load sounds from the content system!!
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 12]
```diff
@@ -21,39 +21,39 @@ As with all software, including releases, there is always a chance for issues an
...
 
 ---
 
 <h2 style="font-weight:bold" align="center">Improvements 🌟</h2>
 
-1. Upgraded all projects in solution to use C# language version **_v10_**  
+1. Upgraded all projects in the solution to use C# language **_v10_**  
    GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
-2. Upgraded all projects in solution to use **_NET 6.0_**  
+2. Upgraded all projects in the solution to use **_NET 6.0_**
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:42]: Make sure there's a period before NET
```suggestion
2. Upgraded all projects in the solution to use **_.NET 6.0_**  
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 13]
```diff
@@ -21,39 +21,39 @@ As with all software, including releases, there is always a chance for issues an
...
+1. Upgraded all projects in the solution to use C# language **_v10_**  
    GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
-2. Upgraded all projects in solution to use **_NET 6.0_**  
+2. Upgraded all projects in the solution to use **_NET 6.0_**  
    GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
-1. Improved constructor argument null checks by implementing a guarding patterns  
+1. Improved constructor argument null checks by implementing a guarding pattern.  
    GitHub Issues: [#153](https://github.com/KinsonDigital/Velaptor/issues/153)
-   - This also resulted in null checks that were not being performed previously
+   - This resulted in null checks that were not being performed previously
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:46]: Add period at end of sentence
```suggestion
   - This resulted in null checks that were not being performed previously.
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md [hunk 14]
```diff
@@ -21,39 +21,39 @@ As with all software, including releases, there is always a chance for issues an
...
 
 <h2 style="font-weight:bold" align="center">Other 👏</h2>
 
-1. Improvements to PR templates such as the code review checklist, grammar, and other misc. items.  This was done on all pull request templates.  
+1. Improvements to PR templates such as the code review checklist, grammar, and other miscellaneous items.  This was done on all pull request templates.  
    GitHub Issues: [#150](https://github.com/KinsonDigital/Velaptor/issues/150), [#158](https://github.com/KinsonDigital/Velaptor/issues/158)
 2. Simple graphical fix with light and dark mode versions of the branching diagram.  
    GitHub Issues: [#159](https://github.com/KinsonDigital/Velaptor/issues/159)
-3. Add a **_BACKERS.md_** file to the root of the solution to hold a list of any backers that  
+3. Added a **_BACKERS.md_** file to the root of the solution to hold a list of any backers that sponsor projects
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md:56]: Add period to end of sentence
```suggestion
3. Added a **_BACKERS.md_** file to the root of the solution to hold a list of any backers that sponsor projects.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — APPROVED** [review]

**Pull request merged**